docs/oid-info.md

```diff
@@ -58,11 +95,90 @@ the git ref that the workflow run was based upon.
 This specifies the username identity in the OtherName Subject Alternative Name, as
 defined by [RFC5280 4.2.1.6](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.6).
 
+
+### 1.3.6.1.4.1.57264.1.8 | Source Repository URI
+
+Should include a fully qualified source repository URL that the build was based
+on. For example: `https://example.com/owner/repository`.
+
+### 1.3.6.1.4.1.57264.1.9 | Source Repository Digest
+
+An immutable reference to a specific version of the source code that the build
+was based upon. For example: `abc123` git commit SHA.
+
+### 1.3.6.1.4.1.57264.1.10 | Source Repository Ref
+
+The source ref that the build run was based upon. For example: `refs/head/main`.
+
+### 1.3.6.1.4.1.57264.1.11 | Source Repository Identifier
+
+The immutable identifier for the source repository the workflow was based upon. For example: `1234` if using a primary key.
+
+### 1.3.6.1.4.1.57264.1.12 | Source Repository Owner URI
+
+Fully qualified URL for the owner of the source repository that the build was based
+on, e.g. `https://example.com/owner`
+
+### 1.3.6.1.4.1.57264.1.13 | Source Repository Owner Identifier
+
+The immutable identifier for the owner of the source repository that the workflow was based upon. For example: `5678` if using a primary key.
+
+### 1.3.6.1.4.1.57264.1.14 | Build Config URI
```

Comment:
To confirm, should this include the server URL? For GH, I see that workflow_ref and job_worflow_ref don't. If so, we should add SHOULD or MUST.

Reply:
Updated these to clarify the url SHOULD be fully qualified.

Just to nail down the GHA details, we're now prefixing these:
- `Run Invocation URI`: `server_url + repository + "/actions/runs/" + run_id + "/attempts/" + run_attempt`
- `Build Signer URI`:  `server_url + job_workflow_ref`

But not these:
- `Source Repository URI`: `repository` (owner/repo)
- `Source Repository Owner URI`: `repository_owner` (owner)
- `Build Config URI`: `workflow_ref`

.. should we be consistent across the board and prefix those with `server_url`? 😅

Reply:
TBH seems ok as is. Should we get this in as is and noodle on specific GHA details when implementing if needed?

Reply:
I'm fine updating in a follow up, but my plan would be to use exactly what is in this doc when implementing for any CI provider (also makes it easier to audit). So if you want to prefix a URI with server_url, we should specify it.